src/modules/jcms_article/src/ArticleCrud.php

```diff
@@ -182,7 +182,11 @@ public function createParagraph(EntityInterface $node, ArticleVersions $articleV
    */
   public function deleteArticle(ArticleVersions $articleVersions) {
```

Comment:
let's keep open a discussion on deleteArticle() btw; we probably should never delete this type of content as Lax won't